lib/awesome_print/formatters/hash_formatter.rb

```diff
@@ -39,10 +39,10 @@ def simple_hash
         "{ #{printable_hash.join(', ')} }"
       end
 
-      def printable_hash
+      def printable_hash # rubocop:disable Metrics/MethodLength
         data = printable_keys
-        width = left_width(data)
 
+        width = left_width(data) if options[:justify_hash]
```

Comment:
width is now no longer set, which will potentially have unexpected consequences in L49, L51 and/or L56.

Reply:
is now *potentially* no longer set. :)